frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -330,6 +393,35 @@ const HostSummary = ({
     ": unavailable"
   );
 
+  const renderDeviceStatusTag = () => {
+    if (!hostMdmDeviceStatus || hostMdmDeviceStatus === "unlocked") return null;
+
+    const tag = DEVICE_STATUS_TAGS[hostMdmDeviceStatus];
+
+    const classNames = classnames(
+      `${baseClass}__device-status-tag`,
+      tag.tagType
+    );
+
+    return (
+      <>
+        <span className={classNames} data-tip data-for="tag-tooltip">
+          {tag.title}
```

Comment:
The Figma has these yellow pill things as all uppercase

Reply:
updated